frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -177,6 +178,12 @@ export default {
         this.debouncedSave()
       }
     },
+    async onBlur() {
+      if (!this.isSaving && this.autoSave) {
+        this.localValue = this.apiValue
+        this.parsedLocalValue = this.parse ? this.parse(this.apiValue) : this.apiValue
+      }
```

Comment:
If I leave an input field before the `debouncedSave()` has triggered a save cycle, then the `localValue` is reset to the original value - and my input is lost.

Try: Page Admin, change camp-name and blur field very quickly -> input is lost.

Reply:
Should I rather remove this functionality, or add a more complicated way to only reset to apiValue if all save requests were successful?